docs/book/user-guide/production-guide/cloud-orchestration.md

```diff
@@ -0,0 +1,166 @@
+---
+description: Orchestrate using cloud resources.
+---
+
+# Orchestrating pipelines on the cloud
+
+Until now, we've only run pipelines locally. The next step is get free from our local machines, and transition our pipelines to execute on the cloud. This will enable you to run your MLOps pipelines in a cloud environment, leveraging the scalability and robustness that cloud platforms offer.
```

Comment:
```suggestion
Until now, we've only run pipelines locally. The next step is to get free from our local machines and transition our pipelines to execute on the cloud. This will enable you to run your MLOps pipelines in a cloud environment, leveraging the scalability and robustness that cloud platforms offer.
```